build.sh

```diff
@@ -66,14 +67,55 @@ function build_msi {
 
   cp build/coreboot.rom ${BOARD}_${FW_VERSION}_$1.rom
   if [ $? -eq 0 ]; then
-    echo "Result binary placed in $PWD/${BOARD}_${FW_VERSION}_$1.rom" 
+    echo "Result binary placed in $PWD/${BOARD}_${FW_VERSION}_$1.rom"
     sha256sum ${BOARD}_${FW_VERSION}_$1.rom > ${BOARD}_${FW_VERSION}_$1.rom.sha256
   else
     echo "Build failed!"
     exit 1
   fi
 }
 
+function build_vp46xx {
+	if [ ! -d 3rdparty/blobs/mainboard ]; then
+		git submodule update --init --checkout
+	fi
+
+	if [ ! -d 3rdparty/blobs/mainboard/protectli/vault_cml ]; then
+		if [ -f protectli_blobs.zip ]; then
+			unzip protectli_blobs.zip -d 3rdparty/blobs/mainboard
+		else
+			echo "Platform blobs missing! You must obtain them first."
+			exit 1
+		fi
+	fi
+
+	version=$(git describe --abbrev=1 --tags --always --dirty)
+	version=${version//protectli_vault_cml_/}
```

Comment:
Try to use this 

```
  DEFCONFIG="configs/config.${BOARD}"
  FW_VERSION=$(cat ${DEFCONFIG} | grep CONFIG_LOCALVERSION | cut -d '=' -f 2 | tr -d '"')
```

Reply:
Done: https://github.com/Dasharo/coreboot/pull/429/commits/8336bf206fd40e63a482e3a6e7cdc4dbe7a0796f